packages/nx-extensions/src/executors/built-script/schema.json

```diff
@@ -10,6 +10,11 @@
 			"description": "Path of the script to run.",
 			"x-prompt": "What script would you like to run?"
 		},
+		"debug": {
+			"type": "boolean",
+			"description": "Use devtools as a debugger.",
+			"x-prompt": "Would you like to use devtools?"
```

Comment:
this is a fairly nondescript prompt. it could help to guide people a bit more into what they may not know is there. something like "Would you like to connect a Node debugging client?"

"devtools" is probably a bit too casual as well, since there are plenty of non-devtools debuggers.

I wonder too if we would always want to use `--inspect-brk` or if there's value in running `--inspect` as well. if we used these, we could open both up and allow a closer correspondence to the Node terminology people may be used to.

Reply:
You're definitely right. Keeping the params consistent would be a good idea. I also plan to add `--trace-warnings`.

Reply:
@dmsnell How does this look? https://github.com/WordPress/playground-tools/pull/110/files#diff-7216652cb6324a396ee2104b76f2e83e8b5e0aa6650acf7e6823d49b7a095078R16